app/main/model/image.py

```diff
@@ -52,11 +54,12 @@ def from_url(url, doc_id, context={}, created_at=None):
     raw = remote_response.read()
     im = Image.open(io.BytesIO(raw)).convert('RGB')
     phash = compute_phash_int(im)
+    sscd = None
```

Comment:
I'm not seeing where, if ever, sscd gets set - am I just missing it?

Reply:
This PR creates the field (nullable) and always set it to null. The next PR should actually set it. My thinking was to break it into two PRs with this first one being "safe" insofar as it just creates the DB column (and always sets it to null)